cmd/init.go

```diff
@@ -77,39 +77,55 @@ var initCmd = &cobra.Command{
 
 		return c.Run()
 	},
-	Long: `Initialize a ksonnet project in a new directory,` + " `app-name`" + `. This process
-consists of two steps:
-
-1. Generating ksonnet-lib. Users can set flags to generate the library based on
-   a variety of data, including server configuration and an OpenAPI
-   specification of a Kubernetes build. By default, this is generated from the
-   capabilities of the cluster specified in the cluster of the current context
-   specified in` + " `$KUBECONFIG`" + `.
-2. Generating the following tree in the current directory.
-
-        app-name/
-          .gitignore     Default .gitignore; can customize VCS
-          .ksonnet/      Metadata for ksonnet
-          environments/
-            default/      [Default generated environment]
-          components/    Top-level Kubernetes objects defining application
-          lib/           user-written .libsonnet files
-          vendor/        part libraries, prototypes
+	Long: `Initialize a ksonnet application in a new directory,` + " `app-name`" + `.
+
+This command will generate all the project scafolding required to begin creating and
+deploying components to Kubernetes clusters.
+
+ksonnet applications are initialized based on your current cluster configurations,
+as defined in your` + " `$KUBECONFIG` " + `environment variable. See examples
+of how to customize configurations in the *Examples* section below.
```

Comment:
rephrase last sentence: ```The *Examples* section below demonstrates how to customize your configurations.```

Reply:
haha re-read the original sentence and realized how awful it was, thanks.